.github/workflows/main.yml

```diff
@@ -0,0 +1,33 @@
+name: Module Builder
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Should this get changed to:
on:push:tags:-'*'

I found this (branches:-main) method will run the build on every push, and update an existing release.
I think we only want to build a release if a Tag is added/created.